packages/gatsby-theme-adr/config/feedOptions.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
           const siteUrl = site.siteMetadata.siteUrl;
           return {
             title: edge.node.frontmatter.title,
-            description: edge.node.frontmatter.deck,
+            description: edge.node.frontmatter.context,
```

Comment:
Shouldn't we also grab `deck` in case a site upgrades but it's still using the old format?